File: LeetCode/Maximum_sum_of_two__subarray_(/2395.py
class Solution:
    def findSubarrays(self, nums: List[int]) -> bool:
        a=set()
        for i in range(0,len(nums)-1):
            sub=nums[i]+nums[i+1]
            if sub in a:
                return True
            a.add(sub)
        return False
        

File: LeetCode/Maximum_sum_of_two__subarray_(/2395.java
class Solution {
    public boolean findSubarrays(int[] nums) {
        Set <Integer> array1= new HashSet<>();
        for (int i=0;i<nums.length-1;i++){
            int sub=nums[i]+nums[i+1];
            if (array1.contains(sub)){
            return true;}
            array1.add(sub);
        }
    return false ;
    }
}